ads/rfp.md

```diff
@@ -0,0 +1,36 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Red for Publishers
+
+## Example
+
+```html
+<amp-ad width="320" height="150"
+  type="rfp"
+  data-adspot-id="XXXXXXXX">
+</amp-ad>
+```
+
+## Configuration
+
+For configuration details and to generate your tags, please contact https://www.fout.co.jp/freakout/contact/.
+
+Supported parameters:
+
+- `data-adspot-id`
```

Comment:
From your code, looks like `data-adspot-id` is required, while `data-sytlesheet-url` and data-country` are optional. Please indicate. You can refer to our [example doc](https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md#required-parameters).

Reply:
Fixed by fbfbfbe